docs/src/literate/tutorial_kalthoff-winkler_dynfrac.jl

```diff
@@ -0,0 +1,91 @@
+# # [Kalthoff-Winkler experiment](@id tutorial_kalthoff-winkler)
+#
+# This tutorial demonstrates how to set up and run the Kalthoff-Winkler experiment using the Peridynamics.jl package.
+# The Kalthoff-Winkler experiment is a classic dynamic fracture experiment involving a pre-notched sample subjected 
+# to impact loading.
+
+# ## Introduction
+#
+# The Kalthoff-Winkler experiment is widely used to study fracture mechanics under high strain rates.
+# This setup provides valuable insights into the behavior of materials under dynamic loading conditions, making it an
+# interesting experiment in the field of fracture mechanics.
+#
+# In this tutorial, we will simulate the Kalthoff-Winkler experiment using peridynamics without the impactor. To do so,
+# first we would import the packages.
+
+# ## Import Packages
+using Peridynamics
+using GLMakie # For visualization
+using LaTeXStrings # For LaTeX string formatting
```

Comment:
I think visualization with ParaView should be sufficient for the moment. Therefore the using statements for `GLMakie` and `LaTeXStrings` can be removed, also because they are not used.

Reply:
Resolved on [fb99d4c]